.github/workflows/pull_request.yaml

```
@@ -15,6 +15,9 @@ permissions:
   contents: read
   actions: write
 
+env:
+  OPENAI_API_KEY: dummuy
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)